### PR TITLE
Issue states

### PR DESCRIPTION
**Pull request by aapomm**
Add support for default issue state. This will be the default state of the issue when uploading from  a plugin.

### CODE REVIEW

File: lib/dradis/plugins/content_service/core.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ module Core
     extend ActiveSupport::Concern
 
     included do
-      attr_accessor :logger, :plugin, :project
+      attr_accessor :default_issue_state, :logger, :plugin, :project
```
Comment (etdsoft) [inline lib/dradis/plugins/content_service/core.rb:6]: Why is this a `Core` attribute?

How is it different from `default_author` or `default_issue_category`?

File: lib/dradis/plugins/content_service/core.rb [hunk 2]
```diff
@@ -12,9 +12,10 @@ module Core
     # @option plugin [Class] the 'wrapper' module of a plugin, e.g.
     #     Dradis::Plugins::Nessus
     def initialize(args={})
-      @logger  = args.fetch(:logger, Rails.logger)
-      @plugin  = args.fetch(:plugin)
-      @project = args[:project]
+      @default_issue_state  = args.fetch(:default_issue_state, :draft)
```
Comment (etdsoft) [inline lib/dradis/plugins/content_service/core.rb:15]: Related to the above.

File: lib/dradis/plugins/content_service/core.rb (hunk 1 above)
Comment (aapomm) [reply lib/dradis/plugins/content_service/core.rb:6]: The difference here is that the `default_issue_state` is a user-defined value that sets the state of all uploaded issues. The `default_author` and `default_issue_category` doesn't seem to be user-defined.

The alternative here is to keep the `default_issue_state` at the Importer level and not pass it to the content service. The downside is, we'd have to make a change in every `content_service.create_issue` call in every upload plugin to pass the `default_issue_state`.

**Comment by aapomm** [comment]
Specs added ✅

**Pull request closed without merging**